### DISCUSSION TITLE
节点钱包balance为0

### DISCUSSION THREAD

**Discussion by jack8086wang** [nervosnetwork/ckb#2410] [General]
1）矿池有转账
2）浏览器可以查到balance
3)节点区块同步已经到最新区块

满足上述3个条件的情况下用./ckb-cli wallet get-capacity -address  xxxxxx查询，balance=0, 请帮忙确认下是什么原因？

**Comment by keroro520** [comment]
ckb-cli 也需要同步完整条链才能查询到最新的状态。加个选项 `--wait-for-sync` 强制 ckb-cli 同步。

`./ckb-cli --wait-for-sync wallet get-capacity -address`

**Comment by jack8086wang** [comment]
加了强制同步可以，bug fixed 谢谢keroro
